Tests/TensorFlowTests/OperatorTests/MatrixTests.swift

```diff
@@ -112,10 +121,23 @@ final class MatrixTests: XCTestCase {
         let expectedGrad = 2 * t3.bandPart(subdiagonalCount: 0, superdiagonalCount: -1)
         XCTAssertEqual(computedGrad, expectedGrad)
     }
+
+    func testEye() {
+        // Test for symmetric identity matrix.
+        var identity: Tensor<Float> = eye(rowCount: 2, columnCount: 2, batchShape:[1])
+        XCTAssertEqual(identity, [[[1.0, 0.0],
```

Comment:
Minor: the trailing `.0` can be omitted from floating-point literals.
```suggestion
        XCTAssertEqual(identity, [[[1, 0],
```

Same below please.